### PR TITLE
feat(component.livecalls): on ovhpabx dashboard display number agent

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/ovhpabx-dashboard-agent-number`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-297
| License          | BSD 3-Clause

## Description

On dashboard for calls statistics, display agent number instead of agent ID.

### CODE REVIEW

File: packages/manager/apps/telecom/src/components/telecom/telephony/alias/liveCalls/telecom-telephony-alias-configuration-liveCalls.component.js [hunk 1]
```diff
@@ -185,6 +185,20 @@ angular.module('managerApp').component('telecomTelephonyAliasLiveCalls', {
...
+                    .v6()
+                    .get({
+                      billingAccount: $stateParams.billingAccount,
+                      serviceName: $stateParams.serviceName,
+                      agentId,
+                    })
+                    .$promise.then((agent) => {
+                      set(agentStatus, 'agentNumber', agent.number);
+                    });
                   return agentStatus;
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/components/telecom/telephony/alias/liveCalls/telecom-telephony-alias-configuration-liveCalls.component.js:202]:
```suggestion
                  set(agentStatus, 'agentId', agentId);

                  // Retrieve agent number
                  return self.apiEndpoint
                    .Hunting()
                    .Agent()
                    .v6()
                    .get({
                      billingAccount: $stateParams.billingAccount,
                      serviceName: $stateParams.serviceName,
                      agentId,
                    })
                    .$promise.then((agent) => ({
                    ...agentStatus, 
                    agentId,
                    agentNumber: agent.number
                    }))
```

**Comment by JeremyDec** [comment]
Hello @Steffy29, this changes look exactly like what I've done for statistics. (See: https://github.com/ovh/manager/pull/3218/files).
Maybe this could be factorize to avoid making it in two places? (I wasn't aware of this before seeing your PR).

**Comment by antleblanc** [comment]
> Hello @Steffy29, this changes looks exactly like what I've done for statistics. (See: https://github.com/ovh/manager/pull/3218/files).
> Maybe this could be factorized to avoid making it in two places? (I wasn't aware of this before seeing your PR).

You're right! It could be factorized.

Since this Pull Request received a `quality check: ok` label, I will merge it to `develop` 
but feel free to open a separate Pull Request to handle this change.

Thank you!

**Comment by antleblanc** [comment]
@Steffy29 I'm taking care of the conflict in order to be able to merge this Pull Request.

**Pull request merged**